Sources/Sentry/SentryOutOfMemoryScopeObserver.m

```diff
@@ -75,15 +75,19 @@ - (void)switchFileHandle
 
 - (void)store:(NSData *)data
 {
-    [self.fileHandle seekToEndOfFile];
-    [self.fileHandle writeData:data];
-    [self.fileHandle writeData:[@"\n" dataUsingEncoding:NSASCIIStringEncoding]];
-
-    self.breadcrumbCounter += 1;
-
-    if (self.breadcrumbCounter >= self.maxBreadcrumbs) {
-        [self switchFileHandle];
-        self.breadcrumbCounter = 0;
+    @try {
+        [self.fileHandle seekToEndOfFile];
+        [self.fileHandle writeData:data];
```

Comment:
We should use the newer one with an inout error in addition to wrapping in try, since this one’s deprecated and the error parameter might catch additional info or be used instead of an exception, which would help performance.

Reply:
Yes, but this one only works on iOS 13.0 and above, see https://developer.apple.com/documentation/foundation/nsfilehandle/3172533-writedata

Reply:
I would create a follow-up issue for using the new APIs, is that fine for you?

Reply:
Sure, that’s fine!